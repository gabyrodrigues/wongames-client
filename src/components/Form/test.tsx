import { FormLink, FormWrapper } from ".";
import { renderWithTheme } from "@utils/tests/helpers";

describe("<Form />", () => {
  it("should render the heading", () => {
    const { container } = renderWithTheme(
      <FormWrapper>
        <FormLink>
          My nice <a href="#">link</a>
        </FormLink>
      </FormWrapper>
    );

    expect(container.parentElement).toMatchSnapshot();
  });
});
