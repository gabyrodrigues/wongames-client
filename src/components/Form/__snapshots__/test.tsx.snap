// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should render the heading 1`] = `
.c0 {
  color: #030517;
  font-size: 1.4rem;
  text-align: center;
}

.c0 a {
  border-bottom: 0.1rem solid #3CD3C1;
  color: #3CD3C1;
  text-decoration: none;
  transition: color,border,0.1s ease-in-out;
}

.c0 a:hover {
  border-bottom: 0.1rem solid #29b3a3;
  color: #29b3a3;
}

<body>
  <div>
    <div
      class=""
    >
      <div
        class="c0"
      >
        My nice 
        <a
          href="#"
        >
          link
        </a>
      </div>
    </div>
  </div>
</body>
`;
